pallets/issue/src/lib.rs

```diff
@@ -377,6 +377,10 @@ impl<T: Config> Pallet<T> {
 	) -> Result<H256, DispatchError> {
 		let amount_requested = Amount::new(amount_requested, vault_id.wrapped_currency());
 
+		// We ensure that the amount requested is compatible with the target chain (ie. it has a
+		// specific amount of trailing zeros)
+		amount_requested.ensure_is_compatible_with_target_chain()?;
```

Comment:
I think we can ignore this because as long as the requested amounts are rounded/compatible, the volume limit will always be rounded as well. And even if it's not, then we don't have a big problem because a volume limit that can only be filled 99.9% and not exactly 100% is still fine (IMO).